sdk/inc/azure/iot/az_iot_provisioning_client.h

```diff
@@ -69,7 +69,7 @@ AZ_NODISCARD az_iot_provisioning_client_options az_iot_provisioning_client_optio
  * @brief Initializes an Azure IoT Provisioning Client.
  *
  * @param[in] client The #az_iot_provisioning_client to use for this call.
- * @param[in] global_device_endpoint The global device endpoint.
+ * @param[in] global_device_hostname The global device endpoint.

```

Comment:
I meant the actual description of the parameter.
```suggestion
 * @param[in] global_device_hostname The global device hostname.
```